proto/p4/config/v1/p4types.proto

```diff
@@ -181,6 +181,17 @@ message StructuredAnnotation {
     ExpressionList expression_list = 2;
```

Comment:
there are quite a few messages in this file with an `annotations` field. Any reason why we are not adding `annotation_locations` for them (@chrispsommers I just noticed that this also applies for `structured_annotations`)?

Reply:
Thanks for pointing that out, I missed those.
I also went ahead and added structured annotations where they were missing.